juniper_official/Solutions/EVPN-VXLAN/spine-evpn-vxlan-kpis.playbook

```diff
@@ -17,7 +17,7 @@
  * 7) Rule vtep-interface-status.rule, detects the vtep interface status
  *    notify anomalies.
  * 8) Rule vtep-interface-traffic.rule,monitors interface traffic and notifies
-      anomalies when traffic exceeds threshold.
+anomalies when traffic exceeds threshold.
```

Comment:
1. Correct the indentation otherwis.e it will impact README auto generate script
2. If i am correct, this playbook upload will fail as * is missing in the beginning of the comment. Please upload this playbook and check

Reply:
Updated